flexport/latitude
# Textarea now measures scrollHeight with rows set to 1

This PR attempts to resolve the bug where Textareas with a dynamic row count would crash IE11 on mount. According to https://html.spec.whatwg.org/multipage/form-elements.html#the-textarea-element, textareas cannot have a row count of zero. This PR updates TextareaInput to not set row count to zero when measuring scrollHeight.

## Manual Testing Plan:
open up the TextareaInput story in storybook,
confirm that dynamic row counts still work by changing min and max row count values

open the TextareaInput story in storybook with IE11,
confirm that dynamic row counts work in IE11 and do not crash the page.
